com.zsmartsystems.zigbee.dongle.ember/src/main/java/com/zsmartsystems/zigbee/dongle/ember/ZigBeeDongleEzsp.java

```diff
@@ -684,7 +684,9 @@ public void sendCommand(final int msgTag, final ZigBeeApsFrame apsFrame) {
         emberApsFrame.setSourceEndpoint(apsFrame.getSourceEndpoint());
         emberApsFrame.setDestinationEndpoint(apsFrame.getDestinationEndpoint());
         emberApsFrame.setSequence(apsFrame.getApsCounter());
-        emberApsFrame.addOptions(EmberApsOption.EMBER_APS_OPTION_RETRY);
+        if (apsFrame.getAckRequest()) {
+            emberApsFrame.addOptions(EmberApsOption.EMBER_APS_OPTION_RETRY);
```

Comment:
Please, can you test that the frame is indeed transmitted with APS Ack Request = FALSE as I don't have a dongle to test with.